doc/UsersGuide/source/HSDcontainer.rst

```diff
@@ -191,20 +190,20 @@ Save the location of the container in an environment variable.
 
 .. code-block:: console
 
-   export img=/path/to/ubuntu22.04-intel-wm-dev-hsd-test.img
+   export img=/path/to/ubuntu22.04-intel-wm-dev-hsd.img
 
 Users may convert a container ``.img`` file to a writable sandbox. This step is optional and unnecessary on most systems (it can take several hours):
 
 .. code-block:: console
 
-   singularity build --sandbox ubuntu22.04-intel-wm-dev-hsd-test $img
+   singularity build --sandbox ubuntu22.04-intel-wm-dev-hsd $img
```

Comment:
please revert back to `singularity build --sandbox ubuntu22.04-intel-wm-dev-hsd-test $img`